frontend/src/app/state/exercise/exercise.selectors.ts

```diff
@@ -17,8 +41,10 @@ export const selectMapImagesTemplates = (state: AppState) =>
     state.exercise.mapImageTemplates;
 export const getSelectMapImageTemplate =
     (mapImageTemplateId: UUID) => (state: AppState) =>
-        state.exercise.mapImageTemplates.find(
-            (template) => template.id === mapImageTemplateId
+        errorIfNotExists(
```

Comment:
I think these should rather return `T | undefined` instead.

Reply:
Are you sure? This would require much work on the usage side of the selectors, I think.

Reply:
An inflationary use of `!` should make this manageable (works in the templates too). But I think it is better to make this assumption on a lower level.

Reply:
On second thought, I would also be fine with using `!` here in the selectors.

Currently this is using defensive programming via the wrapper function and the ugly error checks. I'm against that.

Reply:
I'll replace the `errorIfNotExists` with `!` in the selectors.